lib/matplotlib/projections/__init__.py

```diff
@@ -55,7 +55,15 @@
 from .. import axes, _docstring
 from .geo import AitoffAxes, HammerAxes, LambertAxes, MollweideAxes
 from .polar import PolarAxes
-from mpl_toolkits.mplot3d import Axes3D
+
+try:
+    from mpl_toolkits.mplot3d import Axes3D
+except ImportError:
+    import warnings
+    warnings.warn("Unable to import Axes3D. This may be due to multiple versions of"
+                  "Matplotlib being installed (e.g. as a system package and as a pip"
+                  "package). As a result, the 3D projection is not available.")
```

Comment:
Missing space between words on line breaks:
```suggestion
    warnings.warn("Unable to import Axes3D. This may be due to multiple versions of "
                  "Matplotlib being installed (e.g. as a system package and as a pip "
                  "package). As a result, the 3D projection is not available.")
```